src/main/java/org/gitlab4j/api/DeployKeysApi.java

```diff
@@ -192,6 +192,50 @@ public DeployKey addDeployKey(Object projectIdOrPath, String title, String key,
         return (response.readEntity(DeployKey.class));
     }
 
+    /**
+     * Updates an existing project deploy key.
+     *
+     * <pre><code>GitLab Endpoint: PUT /projects/:id/deploy_keys/:key_id</code></pre>
+     *
+     * @param projectIdOrPath the project in the form of an Integer(ID), String(path), or Project instance
+     * @param deployKey a DeployKey instance with updated properties
+     * @return an updated DeployKey instance
+     * @throws GitLabApiException if any exception occurs
+     */
+    public DeployKey updateDeployKey(Object projectIdOrPath, DeployKey deployKey) throws GitLabApiException {
+
+        final DeployKey key = new DeployKey();
+        key.setCanPush(deployKey.getCanPush());
+        key.setTitle(deployKey.getTitle());
+        final Response response = put(Response.Status.OK, key,
+                "projects", getProjectIdOrPath(projectIdOrPath), "deploy_keys", deployKey.getId());
+
+        return (response.readEntity(DeployKey.class));
+    }
+
+    /**
+     * Updates an existing project deploy key.
+     *
+     * <pre><code>GitLab Endpoint: PUT /projects/:id/deploy_keys/:key_id</code></pre>
+     *
+     * @param projectIdOrPath the project in the form of an Integer(ID), String(path), or Project instance
+     * @param canPush can deploy key push to the project's repository, optional
+     * @param deployKeyId the ID of the deploy key to update
+     * @param title the new deploy key's title, required
+     * @return an updated DeployKey instance
+     * @throws GitLabApiException if any exception occurs
+     */
+    public DeployKey updateDeployKey(Object projectIdOrPath, Integer deployKeyId, Boolean canPush, String title) throws GitLabApiException {
```

Comment:
Please order the parameters in the order requested:
```java
public DeployKey updateDeployKey(Object projectIdOrPath, Integer deployKeyId, String title, Boolean canPush)
```
This is to make it match the GitLab API documentation at:
https://docs.gitlab.com/ee/api/deploy_keys.html#update-deploy-key

And make sure to list ```title``` as optional and use the word "updated" instead of "new", oddly the documentation at the above link has it wrong.

Reply:
Sure, changed.